devops/proxies-apply

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+yarn config set proxy http://proxy:8080
```

Comment:
echo "Applying http_proxy for Yarn..."
...echo "Applying https_proxy for Yarn..."